syntax = "proto3";

package vad;

service VoiceActivityDetector {
    rpc detect(stream DetectRequest) returns (DetectResponse) {}
}

message DetectRequest {
    bytes buffer = 1;
}

message Timestamp {
    float start = 1;
    float end = 2;
}

message DetectResponse {
    repeated Timestamp timestamps = 1;
}
